src/app/services/config.ts

```diff
@@ -0,0 +1,57 @@
+import axios from 'axios';
```

Comment:
this file name is misleading. let's discuss it later, as I've lots of remarks to this axios config